prow/cmd/checkconfig/main.go

```diff
@@ -146,10 +143,13 @@ func main() {
 	cfg := configAgent.Config()
 
 	pluginAgent := plugins.ConfigAgent{}
+	var pcfg *plugins.Configuration
 	if err := pluginAgent.Load(o.pluginConfig); err != nil {
```

Comment:
if `o.pluginConfig` is empty, then `pcfg = nil`, we still want to catch when it fails to load a non-empty plugins config

Reply:
done

Reply:
hummm, I think here should be something like
```
if o.pluginConfig != "" {
  if err := pluginAgent.Load(o.pluginConfig); err != nil {
    Fatal
  }
  pcfg = pluginAgent.Config()
}
```

Reply:
done